### PR TITLE
Test Keccak WordParts

### PR DESCRIPTION
**Pull request by naure**
### Description

This adds unittests for WordParts and its helper functions.

### Issue Link

No issue, continuous improvement.

### Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

### Contents

- Extract a function out of WordParts to highlight its two cases of uniform=true/false.
- Remove arguments "normalize" or "uniform" that are actually constants.
- Show examples of the structure of part sizes and word parts in tests.

### Rationale

What the tested code does and its outputs are not quite obvious.

### How Has This Been Tested?

    cargo test -p zkevm-circuits --features test  -- keccak

### CODE REVIEW

File: zkevm-circuits/src/keccak_circuit.rs [hunk 1]
```diff
@@ -242,7 +240,6 @@ impl<F: Field> SubCircuitConfig<F> for KeccakCircuitConfig<F> {
                 c,
                 1,
                 part_size_c,
-                false,
```
Comment (naure) [inline zkevm-circuits/src/keccak_circuit.rs:245]: `split` without "uniform" layout, constant `false`.

File: zkevm-circuits/src/keccak_circuit.rs [hunk 2]
```diff
@@ -338,7 +335,6 @@ impl<F: Field> SubCircuitConfig<F> for KeccakCircuitConfig<F> {
                     s[i][j].clone(),
                     RHO_MATRIX[i][j],
                     part_size,
-                    true,
```
Comment (naure) [inline zkevm-circuits/src/keccak_circuit.rs:341]: `split_uniform` with "uniform" layout, constant `true`.

**Review by CPerezz — APPROVED** [review]
LGTM! Thanks a lot for including these!!

**Comment by CPerezz** [comment]
Please @naure could you rebase so that we can merge this?

**Comment by naure** [comment]
Merge conflicts resolved (@CPerezz).

**Pull request merged**